doc/src/manual/metaprogramming.md

```diff
@@ -263,29 +263,28 @@ julia> eval(ans)
 julia> ex = :(a + b)
 :(a + b)
 
-julia> eval(ex)
+julia> eval(current_module(), ex)
```

Comment:
I'd say that's a bug in Documenter, shouldn't need to jump through those kind of hoops. I'll see what can be done about it.